pkg/network/node/egressip.go

```diff
@@ -226,7 +230,11 @@ func (eip *egressIPWatcher) getOfflineResult(retrying bool) (map[string]bool, bo
 			continue
 		}
 
-		online := eip.tracker.Ping(node.nodeIP, timeout)
+		dest := node.nodeIP
+		if node.sdnIP != "" {
+			dest = node.sdnIP
```

Comment:
`egressNode`s are only created in `addEgressIP`, and `sdnIP` is always set there, so this should be unnecessary; you can just pass `node.sdnIP` to `Ping`